gui-easy-lib/gui/easy/private/view/input.rkt

```diff
@@ -51,7 +51,7 @@
       (when background-color
         (send the-field set-field-background background-color))
       (begin0 the-field
-        (send the-field set-context 'last-val content)))
+              (send the-field set-context 'last-val content)))
```

Comment:
Meta note: I think you've formatted a bunch of code that didn't need to be touched

Reply:
Ah, hate it when that happens. :/ 

Is there a particular linting set of rules I could use to avoid these problems? 
I mostly use drracket but otherwise a neovim user.

Reply:
I'm not sure: I might have configured begin0 to indent correctly recently, but I'm not sure about case/dep.

Depending on the (Neo)Vim plugins you use, it might be as simple as adding those forms to `lispwords`.